doc/06-helper.md

```diff
@@ -112,6 +112,27 @@ You can check
 the [Symfony documentation](https://symfony.com/doc/current/components/finder.html)
 for more information about this class and how to use it.
 
+## HTTP Client
+
+The `http()` function returns an instance of
```

Comment:
just wondering, `http()` or `http_client()` ?

Reply:
http() is fine for me, client is useless: it's implicit here